tests/aws/services/lambda_/test_lambda_integration_xray.py

```diff
@@ -35,3 +36,40 @@ def test_traceid_outside_handler(create_lambda_function, lambda_su_role, tracing
     assert parsed_result_1["trace_id_outside_handler"] == "None"
     assert parsed_result_2["trace_id_outside_handler"] == "None"
     assert parsed_result_1["trace_id_inside_handler"] != parsed_result_2["trace_id_inside_handler"]
+
+
+@markers.aws.validated
+def test_xray_trace_propagation(
+    create_lambda_function, lambda_su_role, snapshot, aws_client, cleanups
+):
+    """Test trace header parsing and propagation from an incoming Lambda invoke request into a Lambda invocation.
+    This test should work independently of the TracingConfig: PassThrough (default) vs. Active
+    https://stackoverflow.com/questions/50077890/aws-sam-x-ray-tracing-active-vs-passthrough
+    """
+    fn_name = f"test-xray-trace-propagation-fn-{short_uid()}"
+
+    create_lambda_function(
+        func_name=fn_name,
+        handler_file=TEST_LAMBDA_XRAY_TRACEID,
+        runtime=Runtime.python3_12,
+    )
+
+    # add boto hook
+    root_trace_id = "1-3152b799-8954dae64eda91bc9a23a7e8"
+    xray_trace_header = TraceHeader(root=root_trace_id, parent="7fa8c0f79203be72", sampled=1)
```

Comment:
Does this present any issues when testing repeatedly against AWS with the same trace id / parent?

Reply:
nope, works fine.
AWS does not perform any magic validation trying to re-construct the implicit start timestamp encoded here :)

PS: I fixed the repeated AWS validation issues in other X-Ray tests in this ext PR (awaiting review, low prio): https://github.com/localstack/localstack-ext/pull/3596